Updates developer documentation (#1155)
eclipse/buildship: docs/development/Setup.md

![Initial workspace](oomph/install-4.png)

#### Text file encoding

If your are using Windows, make sure the text file encoding is set to `UTF-8`. This can be done under Windows > Preferences > General > Workspace > Text file encoding. Otherwise you might encounter `Unexpected character` errors.

#### Change the target platform

The target platform selected during the installation can be changed, even if it is a bit hidden. Open one of the `tooling-eXXX.target` files and select `Set as Active Target Platform`

> 

Reviewer: I think what is yet missing here is the step to set the `jdk8.location` and `jdk11.location` (in `~/.gradle/gradle.properties`) as per @donat's [comment](https://github.com/eclipse/buildship/pull/1154#pullrequestreview-963599095)

Author: I added a section describing the Gradle setup, including the requirement to use JDK 11 for Gradle.

> 

Reviewer: I think what is yet missing here is the step to set the `jdk8.location` and `jdk11.location` (in `~/.gradle/gradle.properties`) as per @donat's [comment](https://github.com/eclipse/buildship/pull/1154#pullrequestreview-963599095)

![Initial workspace](oomph/install-4.png)

#### Text file encoding

If your are using Windows, make sure the text file encoding is set to `UTF-8`. This can be done under Windows > Preferences > General > Workspace > Text file encoding. Otherwise you might encounter `Unexpected character` errors.

#### Change the target platform

> The target platform selected during the installation can be changed, even if it is a bit hidden. Open one of the `tooling-eXXX.target` files and select `Set as Active Target Platform`

Reviewer 2: ```suggestion
The target platform selected during the installation can be changed. To do that, open one of the `tooling-eXXX.target` files and select `Set as Active Target Platform`
```
